frontend/components/Logo.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
This isn't exactly the logo — it's just a decorator for our titles. Maybe calling it something like TitleDecorator would be more descriptive?